src/components/Stories/Stories.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+
+import {block} from '../utils/cn';
+import {Modal, ModalCloseReason} from '../Modal';
+import {Button} from '../Button';
+import {ButtonClose} from '../Dialog/ButtonClose/ButtonClose';
+import {Link} from '../Link';
+import {MediaRenderer} from './components';
+import {Story} from './types';
+import i18n from './i18n';
+
+import './Stories.scss';
+
+const b = block('stories');
+
+export interface StoriesProps {
+    open: boolean;
+    stories: Story[];
+    onClose?: (
+        event: MouseEvent | KeyboardEvent | React.MouseEvent<HTMLElement, MouseEvent>,
+        reason: ModalCloseReason | 'closeButtonClick',
+    ) => void;
+    startStoryIndex?: number;
+    onPreviousClick?: (storyIndex: number) => void;
+    onNextClick?: (storyIndex: number) => void;
+}
+
+export function Stories({
+    open,
+    onClose,
+    stories,
+    onPreviousClick,
+    onNextClick,
+    startStoryIndex,
+}: StoriesProps) {
+    const startIndex = React.useMemo(() => {
```

Comment:
useMemo is redundant for memoizing numbers

Reply:
Fixed